packages/cubejs-schema-compiler/src/adapter/PreAggregations.js

```diff
@@ -593,18 +599,21 @@ export class PreAggregations {
     return this.query.cubeEvaluator.evaluatePreAggregationReferences(cube, aggregation);
   }
 
-  originalSqlPreAggregationTable(preAggregation) {
-    if (this.canPartitionsBeUsed(preAggregation)) {
-      return this.partitionUnion(preAggregation);
+  originalSqlPreAggregationTable(preAggregationDescription) {
```

Comment:
Might be silly, but should the variable `preAggregationDescription` be called `preAggregationDescriptor` instead?

Reply:
@hassankhan First, I renamed it to descriptor, but I realize that we have a method that called description.

Reply:
Makes sense, thanks for clarifying 👍